chado_tests/test_cvterm.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `harvdev_utils` package."""
+import unittest
+import pytest
+import os
+import time
+import psycopg2
+import subprocess
+
+# Minimal prototype test for new proforma parsing software.
+# SQL Alchemy imports
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from harvdev_utils.chado_functions import (
+    get_cvterm, check_cvterm_has_prop, check_cvterm_is_allowed, CodingError
+)
+conn2 = False
+session = None
+
+def setup_module(module):
+    """ setup any state specific to the execution of the given module."""
+    global conn2, session
+    conn2 = module.conn = startup_db()
+    session = get_session()
+
+def teardown_module(module):
```

Comment:
What's the difference between this "teardown_module()" function and the "stop_db()" function that it calls? They seem redundant

Reply:
Just in case I want to do more stuff on the tar down. (Actually stop_db copied from elsewhere and left as a method as it was, could have been pu in teardown_modile directly but do not bother,